docs/system-design/feature-design/1. Planning/Entity Type Relationship Overlap + Bidirectional Detection.md

```
@@ -0,0 +1,67 @@
+
+## Tasks
+
+- [ ]
+- [ ]
+- [ ]
+
+---
+
+## Notes
+
+_Anything else relevant_
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)